src/app/base/components/GenericTable/GroupRowActions/GroupRowActions.test.tsx

```diff
@@ -0,0 +1,33 @@
+import GroupRowActions from "./GroupRowActions";
+
+import { render, userEvent, screen } from "@/testing/utils";
+
+const getMockRow = (isExpanded: boolean) => {
+  return {
+    getIsExpanded: vi.fn(() => isExpanded),
+    toggleExpanded: vi.fn(),
+  };
+};
+
+it("calls toggleExpanded when button is clicked", async () => {
+  const mockRow = getMockRow(true);
+  // @ts-ignore
```

Comment:
Tried messing around with this locally to see if we could get around this, but I couldn't quite figure it out. If you could just leave a comment here to explain the reasoning, that would be great :+1: